src/main/kotlin/no/nav/syfo/infrastructure/database/repository/POppfolgingsenhet.kt

```diff
@@ -15,10 +16,15 @@ data class POppfolgingsenhet(
     val createdAt: OffsetDateTime,
 )
 
-fun POppfolgingsenhet.toOppfolgingsenhet() = Oppfolgingsenhet(
+fun POppfolgingsenhet.toOppfolgingsenhet(enhetNavn: String?) = Oppfolgingsenhet(
     uuid = this.uuid,
     personident = PersonIdentNumber(this.personident),
-    enhetId = this.oppfolgingsenhet?.let { EnhetId(it) },
+    enhet = this.oppfolgingsenhet?.let {
+        Enhet(
+            enhetId = EnhetId(it),
+            navn = enhetNavn!!,
```

Comment:
Skjønner ikke helt hvordan vi hver gang kan være 100% sikker på at denne har en verdi 🤔 Er det noen mulighet for å gjøre `enhetsNavn` input til ikke optional? 😊
```suggestion
            navn = enhetNavn,
```

Reply:
> Skjønner ikke helt hvordan vi hver gang kan være 100% sikker på at denne har en verdi 🤔 Er det noen mulighet for å gjøre `enhetsNavn` input til ikke optional? 😊

Det jeg har tenkt her er at `enhetNavn` alltid vil ha en verdi når det er satt en oppfølgingsenhet som er forskjellig fra null (men dette navnet må hentes fra Norg). Men det kan godt være det er noe her vi bør tenke litt mer på.

Reply:
Har refaktorert slik at enhetNavn ikke er nullable her.

Reply:
Nice! 🤩